app/build.gradle

```diff
@@ -93,12 +93,12 @@ dependencies {
 }
 
 task installAUT(type: Exec) {
-    group = 'test'
-    description = 'Install app under test'
+    group = "test"
```

Comment:
single quotes are ok. One only uses double quotes in Groovy together with templates